Resources/Prototypes/Body/Parts/base.yml

```diff
@@ -14,6 +14,14 @@
   - type: Damageable
     damageContainer: Inorganic # Shitmed Change
   - type: BodyPart
+  # DeltaV - Specific part thresholds rather than single parent
+  #   severThresholds: # Shitmed change
+  #   - types:
+  #       Piercing: 90
+  #   - types:
+  #       Slash: 90
+  #   - types:
+  #       Blunt: 90
```

Comment:
does nothing remove

Reply:
made with the assumption that this would be ported back to shitmed & uncommented